src/ply.rs

```diff
@@ -27,6 +27,7 @@ use std::str;
 struct Header {
     format: Format,
     elements: Vec<Element>,
+    offset: Vector3<f64>,
```

Comment:
I think in general it may be nicer to have the offset change in a separate PR, since the two are unrelated. What do you think?

Reply:
This is now the offset PR.